tests/cypress/config.js

```diff
@@ -6,7 +6,15 @@ module.exports = defineConfig({
   screenshotsFolder: 'tests/cypress/screenshots',
   videosFolder: 'tests/cypress/videos',
   downloadsFolder: 'tests/cypress/downloads',
-  video: false,
+  video: true,
+  reporter: 'mochawesome',
+  reporterOptions: {
+    mochaFile: "mochawesome-[name]",
+    reportDir: __dirname+"/reports",
```

Comment:
@jayedul Can we hard code the relative path to main consistency? as we are doing in https://github.com/10up/cypress-wp-utils/pull/63/files#diff-c6bacf338af5eaa9a6bd78b3183be542f3b59859910a1c48214c01288b7d8431R13